docs/content/what-is-a-retry-policy.md

```diff
@@ -5,9 +5,9 @@ tags:
   - explanation
 ---
 
-A Retry Policy is collection of attributes that instructs the Temporal Server how to retry a failure of an [Activity Task Execution](#activity-task-execution) or a [Workflow Execution](#workflow-execution).
+A Retry Policy is collection of attributes that instructs the Temporal Server how to retry a failure of a [Workflow Execution](#workflow-execution) or an [Activity Execution](#activity-execution).
```

Comment:
We will need to remember to fix these links at some point -

Reply:
its possible to leave TODOs in the frontmatter, wont matter at all